trading_calendars/exchange_calendar_xphs.py

```diff
@@ -59,6 +59,13 @@ def only_friday(dt):
 
 ChineseNewYearAfter2011 = ChineseNewYear[ChineseNewYear.year > 2011]
 
+PeoplePowerRevolution = Holiday(
```

Comment:
You forgot to add this to `regular_holidays` below.

Reply:
oops, thanks!